core/monitor/price/pricemonitoring_test.go

```diff
@@ -348,230 +343,6 @@ func TestCheckBoundViolationsWithinCurrentTimeWith2HorizonProbabilityPairs(t *te
 	require.False(t, b)
 }
 
-func TestCheckBoundViolationsAcrossTimeWith1HorizonProbabilityPair(t *testing.T) {
```

Comment:
This test effectively didn't test anything (price ranges withing the price monitoring engine were never updated) so I've removed it altogether.